dataline-integrations/singer/postgres/destination/docker/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+function check_connection() {
```

Comment:
Would this be better in `check_connection.py` so all of the check connection logic is isolated to one file and `run.sh` is only responsible for routing?

Reply:
agree with you